src/pages/Admin/EntranceExperience/EntranceExperience.tsx

```diff
@@ -67,6 +67,7 @@ const EntranceExperience: React.FC<EntranceExperienceProps> = ({
       await updateVenue_v2(
         {
           name: venue.name,
+          slug: venue.slug,
```

Comment:
🌟🪱🌟
`spaceSlug`?

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2548/commits/3d30910bba970e8e3052fe6b53a34775a7b5ed05